go/rules-groups-client.go

```diff
@@ -20,6 +20,192 @@ import (
+type ExtractParameters = rulesgroups.ExtractParameters
+
+// RuleMatcher represents a rule matcher.
+type RuleMatcher = rulesgroups.RuleMatcher
+
+// ExtractTimestampParametersFormatStandard represents the standard timestamp format.
+type ExtractTimestampParametersFormatStandard = rulesgroups.ExtractTimestampParameters_FormatStandard
+
+// ApplicationNameConstraint represents an application name constraint.
+type ApplicationNameConstraint = rulesgroups.ApplicationNameConstraint
+
+// RuleMatcherApplicationName represents an application name matcher.
+type RuleMatcherApplicationName = rulesgroups.RuleMatcher_ApplicationName
+
+// SubsystemNameConstraint represents a subsystem name constraint.
+type SubsystemNameConstraint = rulesgroups.SubsystemNameConstraint
+
+// RuleMatcherSubsystemName represents a subsystem name matcher.
+type RuleMatcherSubsystemName = rulesgroups.RuleMatcher_SubsystemName
+
+// SeverityConstraint represents a severity constraint.
+type SeverityConstraint = rulesgroups.SeverityConstraint
+
+// SeverityConstraintValue represents a severity constraint.
+type SeverityConstraintValue = rulesgroups.SeverityConstraint_Value
+
+// JSONExtractParametersDestinationField represents a JSON extract destination field.
+type JSONExtractParametersDestinationField = rulesgroups.JsonExtractParameters_DestinationField
+
+// RuleMatcherSeverity represents a severity matcher.
+type RuleMatcherSeverity = rulesgroups.RuleMatcher_Severity
+
+// RuleGroup represents a rule group.
+type RuleGroup = rulesgroups.RuleGroup
+
+// RuleSubgroup represents a subgroup of a rule.
+type RuleSubgroup = rulesgroups.RuleSubgroup
+
+// Rule represents a rule.
+type Rule = rulesgroups.Rule
+
+// SeverityConstraintValueDebugOrUnspecified represents an unspecified severity or debug level.
+const SeverityConstraintValueDebugOrUnspecified = rulesgroups.SeverityConstraint_VALUE_DEBUG_OR_UNSPECIFIED
+
+// SeverityConstraintValueVerbose represents the verbose severity level.
+const SeverityConstraintValueVerbose = rulesgroups.SeverityConstraint_VALUE_VERBOSE
+
+// SeverityConstraintValueInfo represents the info severity level.
+const SeverityConstraintValueInfo = rulesgroups.SeverityConstraint_VALUE_INFO
+
+// SeverityConstraintValueWarning represents the warning severity level.
+const SeverityConstraintValueWarning = rulesgroups.SeverityConstraint_VALUE_WARNING
+
+// SeverityConstraintValueError represents the error severity level.
+const SeverityConstraintValueError = rulesgroups.SeverityConstraint_VALUE_ERROR
+
+// SeverityConstraintValueCritical represents the critical severity level.
+const SeverityConstraintValueCritical = rulesgroups.SeverityConstraint_VALUE_CRITICAL
+
+// JSONExtractParametersDestinationFieldCategoryOrUnspecified represents the default JSON extract destination field.
+const JSONExtractParametersDestinationFieldCategoryOrUnspecified = rulesgroups.JsonExtractParameters_DESTINATION_FIELD_CATEGORY_OR_UNSPECIFIED
```

Comment:
Personally I like it better when it's more compact and readable with a less repetitive docstring.
~~~go 
// JSONExtract values.
const (
 JSONExtract....
 JSONExtractOther....
 ~~~

Reply:
Done